primer/src/Primer/App/Base.hs

```diff
@@ -54,21 +63,63 @@ data NodeType = BodyNode | SigNode
   deriving (FromJSON, ToJSON) via PrimerJSON NodeType
   deriving anyclass (NFData)
 
--- | Describes what interface element the user has selected.
--- A definition in the left hand nav bar, and possibly a node in that definition.
+-- | Describes which element of a (type or term) definition the student has selected.
+-- We have the following invariant: when this contains a `NodeSelection` with @nodeType = SigNode@,
+-- or any `TypeDefConsFieldSelection`, then they will always have @meta = Right _@.
```

Comment:
Markup seems odd here. `s/`/@/g`?

Reply:
I think the backtick is fine, isn't it? We use it the same way elsewhere. And the `@`'s are there (actually just moved from another file) because they enclose a compound expression, rather than a single identifier.